usr/lib/webapp-manager/common.py

```diff
@@ -246,7 +250,7 @@ def create_webapp(self, name, url, icon, category, browser, custom_parameters, i
             desktop_file.write("[Desktop Entry]\n")
             desktop_file.write("Version=1.0\n")
             desktop_file.write("Name=%s\n" % name)
-            desktop_file.write("Comment=%s\n" % _("Web App"))
+            desktop_file.write("Comment=%s\n" % gettext.gettext("Web App"))
```

Comment:
This shouldn't be necessary:
https://github.com//linuxmint/webapp-manager/blob/master/usr/lib/webapp-manager/common.py#L50